js/newUser.js

```diff
@@ -0,0 +1,29 @@
+function matchPassword() {
+    const password = document.getElementById('password');
+    const reconfirmPassword = document.getElementById('reconfirm-password');
+
+    if (password.value == reconfirmPassword.value) {
+        reconfirmPassword.setCustomValidity('')
+    } else {
+        reconfirmPassword.setCustomValidity('Password do not match')
+    }
+}
+
+const form = document.getElementById('form')
+
+try {
```

Comment:
the try catch block should go inside the addEventListener function, in order to trigger is an error is found inside the function, not while trying to add an event listener to the form